keyboards/angel64/keymaps/default/keymap.c

```diff
@@ -55,4 +55,8 @@ void led_set_user(uint8_t usb_led) {
 }
 
 void keyboard_post_init_user(void) {
+#ifdef RGBLIGHT_ENABLE
```

Comment:
I would suggest removing the default rgblight initialisation here, and instead add `RGB_x` keycodes to the keymap.